### DISCUSSION TITLE
What are these markers?

### DISCUSSION THREAD

**Discussion by RussKie** [gitextensions/gitextensions#12025] [Q&A] [answered]
@gerhardol @mstv what are these markers?
![image](https://github.com/user-attachments/assets/42d89b2a-6764-44a5-b4bc-b1be63f58692)

- Git Extensions 5.1.0.17960
- Build f9e558c2f1645008b4c426f847cadbf10c92fcd0
- Git 2.46.0.windows.1

**Comment by gerhardol** [comment]
Where changes were "inserted" in the add. Remember that there were an empty irrelevant space before 5.1?

**Reply by gerhardol** [reply, marked as answer]
You see that better here:
![image](https://github.com/user-attachments/assets/b57fc485-7862-4e06-a9af-a12c404f4920)
